Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -457,27 +457,34 @@ def test_module(client):
     return 'ok'
 
 
-def list_incidents_command(client, args, is_fetch_incidents=False):
+def list_incidents_command(client, args, is_fetch_incidents=False, first_fetch=False):
+    demisto.debug("starting the list incidents command")
     filter_expression = args.get('filter')
     limit = None if is_fetch_incidents else min(200, int(args.get('limit')))
     next_link = args.get('next_link', '')
+    result = None
+    params = {}
+    demisto.debug(f"is_fetch_incidents:{is_fetch_incidents}, first_fetch:{first_fetch}")
```

Comment:
```suggestion
    demisto.debug(f"{is_fetch_incidents=}, {first_fetch=}")
```
Sweet sweet python 3.8 syntax sugaring